web-components/src/components/input/Input.ts

```diff
@@ -443,6 +456,7 @@ export namespace Input {
                 class="md-input__icon-clear"
                 name="clear-active_12"
                 aria-label=${this.clearAriaLabel || "Clear Input"}
+                tabindex="1"                
```

Comment:
why we need tabindex for md-icon as md-button is itself a focusable element

Reply:
we will be needing this to specify the order of focus. since after pressing tab on input, the focus gets lost and on second click only it goes to the button. so we will be needing tab index to fix that

Reply:
this will result in button and icon both getting the focus, which would not be desirable